CONTRIBUTING.md

```diff
@@ -45,22 +45,119 @@ To add a new plugin you need to do the following.
+| **settingsOptions**        | Options for the plugin's settings.                                                                                                                                                                           | No       | `{}`            |
+| **settingsExample**        | An example configuration for the plugin's settings.                                                                                                                                                          | No       | `null`          |
+| **settingsDescription**    | A description of the settings provided to the `require('${luaName}')${setup}` function.                                                                                                                      | No       | `"Options provided to the require('${luaName}')${setup} function."` |
+| **hasSettings**            | Indicating whether the plugin has settings.                                                                                                                                                                  | No       | `true`          |
+| **extraOptions**           | Additional options for the plugin.                                                                                                                                                                           | No       | `{}`            |
+| **luaName**                | The Lua name for the plugin.                                                                                                                                                                                 | No       | `name` parameter |
+| **setup**                  | The setup function for the plugin.                                                                                                                                                                           | No       | `".setup"`      |
+| **extraConfig**            | Additional configuration for the plugin.                                                                                                                                                                     | No       | `{}`            |
+| **extraPlugins**           | Extra plugins to include.                                                                                                                                                                                    | No       | `[]`            |
+| **extraPackages**          | Extra packages to include.                                                                                                                                                                                   | No       | `[]`            |
+| **callSetup**              | Indicating whether to call the setup function.                                                                                                                                                               | No       | `true`          |
```

Comment:
```suggestion
| **callSetup**              | Indicating whether to call the setup function. Useful when the `setup` function needs to be called in a special way.                                                                                                                                                              | No       | `true`          |
```

Reply:
This was moved up to line 62, but I think it'd still be nice to add the `Useful when the `setup` function needs to be called in a special way.` suggestion?